v2/tools/generator/internal/codegen/pipeline/add_cross_resource_references.go

```diff
@@ -387,5 +388,22 @@ func newKnownReferencesMap(configuration *config.Configuration) map[referencePai
 			typeName: astmodel.MakeTypeName(authorization20200801, "RoleAssignmentProperties"),
 			propName: "RoleDefinitionId",
 		}: true,
+		// EventHub -- TODO: Validate these
```

Comment:
Have you validated these?

Reply:
Yep! Thanks for the catch, removed the comment.